pkg/pcrbruteforcer/reproduce_expected_pcr0_test.go

```diff
@@ -113,48 +127,60 @@ func BenchmarkReproduceExpectedPCR0(b *testing.B) {
 			}
 			measurements, _, _, err := pcr.GetMeasurements(firmware, 0, measureOptions...)
 			if err != nil {
-				b.Fatal(err)
+				panic(err)
 			}
 
 			b.Run("correctPCR0", func(b *testing.B) {
 				b.ReportAllocs()
 				b.ResetTimer()
 				for i := 0; i < b.N; i++ {
-					_, _, _, _ = ReproduceExpectedPCR0(
+					_, err := ReproduceExpectedPCR0(
 						ctx,
 						pcr0Correct,
 						pcr.FlowIntelCBnT0T,
 						measurements,
 						firmware.Buf(),
+						settings,
 					)
+					if err != nil {
+						panic(err)
```

Comment:
Do you want me to cite "Panic at the disco" lyrics here, because that's the way to push me towards citing "Panic at the disco" lyrics :D

Reply:
Fixed.